### PR TITLE
Self Hosting new MaterialIcons fonts

### PR DESCRIPTION
**Pull request by TomFreudenberg**
Hi @Chun-Yang 

here is my PR as discussed in #5 

I have (re-)added also the deprecated Material-Design-Icons fonts to be compatible as long as materializecss project will also drop them.

In addition I added some rules like spin and rotate and renamed all to "md-" prefix.

Looking forward to your comments.

P.S.: Some of the extended rules like md-border are not yet fully ok. I just have taken their definition in this first shot.

### CODE REVIEW

File: bower_components/materialize/sass/components/_material-icons.scss [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+    // Rules for using icons as black on a light background.
+    &.md-dark { color: rgba(0, 0, 0, 0.54); }
+    &.md-dark.md-inactive { color: rgba(0, 0, 0, 0.26); }
+
+    // Rules for using icons as white on a dark background.
+    &.md-light { color: rgba(255, 255, 255, 1); }
+    &.md-light.md-inactive { color: rgba(255, 255, 255, 0.3); }
+
+    /* rules taken from deprecated material-design-icons */
```
Comment (Chun-Yang) [inline bower_components/materialize/sass/components/_material-icons.scss:56]: Do we need these deprecated rules?
I think we should delete the previous icons and use the new ones. If people want to use previous icons (0.96.1), they should use a previous version of meteor-materialize-sass. 
What do you think?

File: bower_components/materialize/sass/components/_material-icons.scss [hunk 2]
```diff
@@ -0,0 +1,194 @@
+$font-material-icons-name   : 'Material Icons';
+$font-material-icons-file   : 'MaterialIcons-Regular';
+
+@font-face {
+    font-family: "#{$font-material-icons-name}";
+    font-weight: 400;
+    font-style: normal;
+    src: url("#{$icons-font-path}#{$font-material-icons-file}.eot"); /* For IE6-8 */
```
Comment (Chun-Yang) [inline bower_components/materialize/sass/components/_material-icons.scss]: $icons-font-path is not defined?

File: package.js [hunk 3]
```diff
@@ -19,6 +19,15 @@ Package.onUse(function(api) {
   // font files
   var fontDirectory = 'bower_components/materialize/';
   var fontFiles = [
+    'font/material-design-icons/Material-Design-Icons.eot',
```
Comment (Chun-Yang) [inline package.js:22]: Material-Design-Icons are not used and against materializecss doc. We should not include them any more.

Comment (TomFreudenberg) [inline package.js:22]: At least they are not yet deprecated in materializecss and they deliver them still. So in this case that also the mdi-\* rules are still in the css I would prefer for backward compatibility to deliver the font until materializecss drop it too

File: bower_components/materialize/sass/components/_material-icons.scss (hunk 1 above)
Comment (TomFreudenberg) [inline bower_components/materialize/sass/components/_material-icons.scss:56]: This are NEW rules from google (see Paragrah: Styling icons in material design in https://google.github.io/material-design-icons/) 

The rules AFTER comment are taken from materialize-sass like SPIN, ROTATE etc. and they are mostly useful - imho

Comment (Chun-Yang) [inline bower_components/materialize/sass/components/_material-icons.scss:56]: Thank you for your explanation.

File: package.js (hunk 3 above)
Comment (Chun-Yang) [inline package.js:22]: OK, I agree.

**Comment by Chun-Yang** [comment]
Thank you! :beers:

**Comment by TomFreudenberg** [comment]
You too for acepting and providing this package!

**Pull request merged**